public/css/module.less

```diff
@@ -6,6 +6,27 @@
 @cert-segment-color-3: #1982C4;
 @cert-segment-color-4: #6A4C93;
 
+.controls.default-layout > :not(:only-child) {
+  // Reset the margin set in ipl web!
```

Comment:
Why the reset? Does it make sense to transfer this to `ipl-web`?

Reply:
Nop! I have already reverted this change on my end!